tools/testsys/src/migration.rs

```diff
@@ -0,0 +1,95 @@
+use crate::crds::{MigrationDirection, MigrationInput};
+use crate::error::{self, Result};
+use bottlerocket_types::agent_config::MigrationConfig;
+use maplit::btreemap;
+use model::Test;
+use snafu::OptionExt;
+
+/// Create a CRD for migrating Bottlerocket instances using SSM commands.
```

Comment:
I'd prefer to keep this in `aws_resources.rs` to signify that this migration is performed using aws tools.

Reply:
I thought about this a little bit more and I think it makes sense to separate out the migration CRD construction from `aws_resources` since the migration CRD is for a test CRD and the test itself is not limited to just run on EKS/ECS. Sure it may use AWS tools but we also use AWS tools in VMware VMs and cluster resource as well.